src/lib.rs

```diff
@@ -296,6 +296,9 @@ const PAYLOAD_MIN_LEN: usize = 20;
 
 const MAX_AMPLIFICATION_FACTOR: usize = 3;
 
+// The maximum number of tracked packet number ranges that need to be acked.
+const MAX_ACK_RANGES: usize = 68;
```

Comment:
Where is `68` coming from? It would be good to add how this number is chosen.

Reply:
It's an estimate of how ack gaps we can fit in a packet, though the calculation it's not super precise (e.g. quicly uses 64 as limit, but that seems too low). I updated the comment a bit to be more clear.

Reply:
Also, we can tweak the number later if we see that it is too high/low.